_locales/cs/pxt-TPBot-strings.json

```
@@ -0,0 +1,44 @@
+{
+  "TPBot.DriveDirection.Backward|block": "vzad",
+  "TPBot.DriveDirection.Forward|block": "vpřed",
+  "TPBot.DriveDirection.Left|block": "vlevo",
+  "TPBot.DriveDirection.Right|block": "vpravo",
+  "TPBot.LineSide.Left|block": "levý",
+  "TPBot.LineSide.Right|block": "pravý",
+  "TPBot.LineState.Black|block": "černou",
+  "TPBot.LineState.White|block": "bílou",
+  "TPBot.MbEvents.Black|block": "černou",
+  "TPBot.MbEvents.White|block": "bílou",
+  "TPBot.MbPins.Left|block": "levý",
+  "TPBot.MbPins.Right|block": "pravý",
+  "TPBot.ServoList.S1|block": "S1",
+  "TPBot.ServoList.S2|block": "S2",
+  "TPBot.ServoList.S3|block": "S3",
+  "TPBot.ServoList.S4|block": "S4",
+  "TPBot.SonarUnit.Centimeters|block": "cm",
+  "TPBot.SonarUnit.Inches|block": "palcích",
+  "TPBot.Sonarjudge.Greater|block": ">",
+  "TPBot.Sonarjudge.Less|block": "<",
+  "TPBot.TrackingState.L_R_line|block": "● ●",
+  "TPBot.TrackingState.L_R_unline|block": "◌ ◌",
+  "TPBot.TrackingState.L_line_R_unline|block": "● ◌",
+  "TPBot.TrackingState.L_unline_R_line|block": "◌ ●",
+  "TPBot.headlightClose|block": "vypni světlomety",
+  "TPBot.headlightColor|block": "nastav barvu světlometů na $color",
+  "TPBot.headlightRGB|block": "nastav barvu světlometů složenou z červené:%r zelené:%g modré:%b",
+  "TPBot.setServo180|block": "pro 180° servo %servo nastav úhel %angle °",
+  "TPBot.setServo360|block": "pro 360° servo %servo nastav rychlost %speed \\%",
+  "TPBot.setServo|block": "nastav úhel %servoType serva %servo na %angle °",
+  "TPBot.setTravelSpeed|block": "jeď %direc rychlostí %speed\\%",
+  "TPBot.setTravelTime|block": "jeď %direc rychlostí %speed\\% po dobu %time sekund",
+  "TPBot.setWheels|block": "nastav rychlost levého kola na %lspeed\\%| a rychlost pravého kola na %rspeed\\%",
```

I would shorten the text a bit to avoid excessive block length.
```suggestion
  "TPBot.setWheels|block": "nastav rychlost levého kola na %lspeed\\%| a pravého kola na %rspeed\\%",
```

or even shorter version

```suggestion
  "TPBot.setWheels|block": "nastav rychlost levého kola na %lspeed\\%| a pravého na %rspeed\\%",
```

---

@lobodpav 

Hmm, I agree with the "je zavolána" -> "spustí se" change. It really sounds more approachable. I'll add that change into "TPBot v2" updates.

When it comes to the long translations (hence long blocks), it's always a difficult decision - is it OK to sacrifice readability for brevity? Your point is good, I just need to think about it a bit... (maybe there's a more concise way).

Thanks for the feedback!

---

Yeah, it's difficult to find good wording. I am a bit sensitive to wide blocks because my kids use tablets for programming - free space is a very precious resource there 😊

Anyway, thanks for your great translations! I can see your name on that PR for the DFRobot Maqueen Plus, which I also ordered.